frontend/packages/console-app/locales/en/console-app.json

```diff
@@ -223,13 +223,22 @@
   "Terminal {{number}}": "Terminal {{number}}",
   "Close terminal tab": "Close terminal tab",
   "Add new tab": "Add new tab",
+  "This Project will be used to initialize your command line terminal": "This Project will be used to initialize your command line terminal",
   "Initialize terminal": "Initialize terminal",
+  "Click on the names to access advanced options for <0></0>.": "Click on the names to access advanced options for <0></0>.",
+  "Timeout": "Timeout",
+  "Image": "Image",
   "Start": "Start",
   "Cancel": "Cancel",
+  "Set custom image for the terminal.": "Set custom image for the terminal.",
   "Select Project": "Select Project",
   "Create Project": "Create Project",
-  "This Project will be used to initialize your command line terminal": "This Project will be used to initialize your command line terminal",
   "Project name": "Project name",
+  "Seconds": "Seconds",
+  "Minutes": "Minutes",
+  "Hours": "Hours",
+  "Miliseconds": "Miliseconds",
```

Comment:
`s/Miliseconds/Milliseconds/`

Reply:
Thaks @amisevsk, here is the PR to get this right https://github.com/openshift/console/pull/12423